recipes/npy2bdv/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "npy2bdv" %}
+{% set version = "1.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Using GitHub rather than PyPI since the LICENSE file is missing there
+  url: https://github.com/nvladimus/npy2bdv/archive/refs/tags/v.1.0.8.tar.gz
```

Comment:
```suggestion
  url: https://github.com/nvladimus/npy2bdv/archive/refs/tags/v.{{ version }}.tar.gz
```

Reply:
The reason I didn't do this is because the other release URL has `v1.0.0`, and so I think that leading dot is a mistake that won't be repeated. Happy to change it if it matters though.

Reply:
It does for the bot and consistency with the declared version above, otherwise they can easily diverge.

That is why we prefer stable URLs. You can still use PyPI's URL here and add the license manually.

Reply:
Happy to do that, but I don't know how

Reply:
You can copy the license into the recipe folder. The build will find it, based on the name you add in the recipe, and ship it in the package.